### DISCUSSION TITLE
Database is not editable when I use the Blue Topaz Theme

### DISCUSSION THREAD

**Discussion by huenchen** [RafaelGB/obsidian-db-folder#550] [Q&A] [answered]
Whenever I use the Blue Topaz Theme, I cannot write anything into the database. Everything else works fine. 
I tried several databases. 
Whenever I switch back to the Standard Theme it works again. Does anyone know a workaround or has expirienced the same?

**Comment by RafaelGB** [comment, marked as answer]
Since the 1.0 of Obsidian, all the themes needs adaptations. I can not offer support with all the themes right now, just with the default one.

Once the versions looks more stable we will be able to solve this issue

**Reply by huenchen** [reply]
Thanks a lot for your quick answer! So I will always use the Standard Theme when working with the Databases. This is no big deal.

**Comment by huenchen** [comment]
Only a short hint, The “Minimal – Theme” is working pretty good with dbFolder too.

 

Liebe Grüße,

 

Holger  

 

Von: Rafael Gómez Bermejo ***@***.***>
Antworten an: RafaelGB/obsidian-db-folder ***@***.***>
Datum: Samstag, 22. Oktober 2022 um 22:48
An: RafaelGB/obsidian-db-folder ***@***.***>
Cc: Holger ***@***.***>, Author ***@***.***>
Betreff: Re: [RafaelGB/obsidian-db-folder] Database is not editable when I use the Blue Topaz Theme (Discussion #550)

 

Since the 1.0 of Obsidian, all the themes needs adaptations. I can not offer support with all the themes right now, just with the default one.

Once the versions looks more stable we will be able to solve this issue
